app/assets/javascripts/constants/image_constants.es6.jsx.erb

```diff
@@ -30,6 +30,16 @@
         clarinet: "<%= asset_path("instruments/clarinet.png") %>",
       }
     }
+
+    get lesson_card() {
+      return {
+        cancel: "<%= asset_path("lesson_card/cancel.png") %>",
+        location: "<%= asset_path("lesson_card/location.png") %>",
+        modify: "<%= asset_path("lesson_card/modify.png") %>",
+        person: "<%= asset_path("lesson_card/person.png") %>",
+        time: "<%= asset_path("lesson_card/time.png") %>",
+      }
```

Comment:
can you add a semicolon here? I also realized I didn't put semicolons for the other ones, so can you just add it for all of them :O ty ty